src/sentry/models/options/option.py

```diff
@@ -54,9 +54,9 @@ def query_for_relocation_export(cls, q: models.Q, pk_map: PrimaryKeyMap) -> mode
         return q & ~models.Q(
             key__in={
                 "sentry:install-id",  # Only used on self-hosted
+                "sentry:latest_version",  # Auto-generated periodically, which defeats comparison
```

Comment:
why the change in order? does it have any difference or is it just formatting alphabetical thing?

Reply:
No difference, just realized I alphabetized incorrectly when I did in the previously mentioned PR.